CONTRIBUTING.md

```
@@ -72,6 +72,21 @@ If you haven't already, please read the main [README.md](README.md) for an overv
   - [Integration Examples](#integration-examples)
   - [TypeScript Types](#typescript-types)
   - [Benefits](#benefits-1)
+- [GDPR Cookie Consent Implementation](#gdpr-cookie-consent-implementation)
+  - [Features](#features-1)
+  - [Cookie Categories](#cookie-categories)
+  - [Files Added/Modified](#files-addedmodified)
+  - [Usage](#usage-1)
+  - [Legal Compliance](#legal-compliance)
+  - [Customization](#customization)
+  - [Testing](#testing-1)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)